diff --git a/app/code/Magento/Checkout/Service/V1/Data/PaymentMethod/Converter.php b/app/code/Magento/Checkout/Service/V1/Data/PaymentMethod/Converter.php
index 2847b7e..e8f0939 100644
--- a/app/code/Magento/Checkout/Service/V1/Data/PaymentMethod/Converter.php
+++ b/app/code/Magento/Checkout/Service/V1/Data/PaymentMethod/Converter.php
@@ -5,20 +5,26 @@
  * @copyright   {copyright}
  * @license     {license_link}
  */
-
 namespace Magento\Checkout\Service\V1\Data\PaymentMethod;
 
 use Magento\Checkout\Service\V1\Data\PaymentMethod as QuotePaymentMethod;
 
+/**
+ * Payment method converter.
+ */
 class Converter
 {
     /**
+     * Payment method builder.
+     *
      * @var \Magento\Checkout\Service\V1\Data\Cart\PaymentMethodBuilder
      */
     protected $builder;
 
     /**
-     * @param \Magento\Checkout\Service\V1\Data\PaymentMethodBuilder $builder
+     * Constructs a payment method converter object.
+     *
+     * @param \Magento\Checkout\Service\V1\Data\PaymentMethodBuilder $builder Payment method builder.
      */
     public function __construct(\Magento\Checkout\Service\V1\Data\PaymentMethodBuilder $builder)
     {
@@ -26,10 +32,10 @@ class Converter
     }
 
     /**
-     * Convert quote payment object to payment data object
+     * Converts quote payment object to payment data object.
      *
-     * @param \Magento\Payment\Model\MethodInterface $object
-     * @return \Magento\Checkout\Service\V1\Data\Cart\PaymentMethod
+     * @param \Magento\Payment\Model\MethodInterface $object The quote payment object.
+     * @return \Magento\Checkout\Service\V1\Data\Cart\PaymentMethod Payment data object.
      */
     public function toDataObject(\Magento\Payment\Model\MethodInterface $object)
     {
